src/main/java/org/sefglobal/scholarx/service/SimpleJavaMail.java

```diff
@@ -0,0 +1,30 @@
+package org.sefglobal.scholarx.service;
+
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.email.EmailBuilder;
+import org.simplejavamail.mailer.MailerBuilder;
+
+public class SimpleJavaMail {
+
+    public void sendEmail(String fname,String emailAddress, String subject, String message, boolean showButton) {
+
+        String username = "pasindur2@gmail.com";
+        String password = "wwooravkrkhorobx";
+        String host = "smtp.mailtrap.io";
+        int port = 2525;
+
+        Email email = EmailBuilder.startingBlank()
+                .from("Sustainable Education Foundation", username)
+                .to(fname, emailAddress)
+                .withSubject(subject)
+                .withHTMLText(message)
+                .buildEmail();
+
+        MailerBuilder
+                .withSMTPServer(host, port, username, password)
+                .buildMailer()
+                .sendMail(email);
```

Comment:
This will create mail builder instances whenever we send an email; which involves logging in using provided credentials whenever we send an email (That's the issue we are trying to solve). We should use a single instance Instead.

Reply:
cc: @pasindur99

Reply:
I think the issue will be solved with [f8c7d2f](https://github.com/sef-global/scholarx/pull/345/commits/f8c7d2f75c9ad5435189358274bf19970a029980). 
But I have no idea how to use those credentials as environment variables. Can you suggest a way to fix it?
@jayasanka-sack

Reply:
This might help if we are keeping credentials as properties:

https://www.baeldung.com/configuration-properties-in-spring-boot

@Piumal1999 are we going to keep them as properties, or environment variables?

Reply:
Let's keep them as properties. 

> This might help if we are keeping credentials as properties:
https://www.baeldung.com/configuration-properties-in-spring-boot


Yeah.

FYI we have done something similar when creating the authService. But it may not be the optimum way. https://github.com/sef-global/scholarx/blob/72672220946e4fb4fab5a440e84e41ed058d3f0e/src/main/java/org/sefglobal/scholarx/service/AuthUserService.java#L57